### ISSUE TITLE
KID not found (error 205)

### ISSUE THREAD

**Issue by dbradmit** [keybase/client#7104]
I am trying to login to the keybase app and receive the same error across Windows, macOS, and Android: **KID not found (error 205)**

The error occurs in the app as well as CLI.

Steps to reproduce:

1. I enter my username 
2. I select my paper key
3. I enter my paper key
4. Presented with _KID not found (error 205)_

or

1. I enter my email address 
2. I enter my passphrase
3. I select my paper key
4. I enter my paper key
5. Presented with _KID not found (error 205)_

my log id: 657a0d0af8e7ce05d86e0f1c

**Comment by maxtaco** [comment]
What are the first two words of your paper key? (They are public)

**Comment by maxtaco** [comment]
@patrickxb  can you check this one out?

**Comment by dbradmit** [comment]
The first two words of the paper key are "pact panic".

**Comment by patrickxb** [comment]
Yes, I'll look

**Comment by patrickxb** [comment]
@dbradmit I'm sure you've tried this, but can you double-check that you are entering the correct paper key?  From the log, it looks like the paper key you are entering appears to be valid (we do some sanity-checking on the input), but it is not matching the `pact picnic` one in your account.

**Comment by patrickxb** [comment]
@dbradmit ah, I just read your comment.  Looks like you got it working.

**Comment by maxtaco** [comment]
Still that error message is pretty rought.

**Comment by dbradmit** [comment]
Yeah, the error message is rough but my issue is resolved.

Thanks!

**Comment by maxtaco** [comment]
@patrickxb can you make a ticket to fix the error experience here? thanks!

**Comment by patrickxb** [comment]
@maxtaco Yes.  Surprising thing is that it was a valid paper key.  Current error messages handle most typos...

**Comment by maxtaco** [comment]
cool thanks
